design-tokens/tokens/layer1/base.json

```diff
@@ -113,78 +133,150 @@
       "type": "letterSpacing"
     },
     "increased": {
-      "value": "150%",
+      "value": "2%",
       "type": "letterSpacing"
     },
     "decreased": {
-      "value": "-5%",
+      "value": "-2%",
       "type": "letterSpacing"
     }
   },
   "paragraphSpacing": {
-    "h1": {
-      "value": "32",
+    "small": {
+      "value": "1rem",
       "type": "paragraphSpacing"
     },
-    "h2": {
-      "value": "26",
+    "medium": {
+      "value": "1.125rem",
+      "type": "paragraphSpacing"
+    },
+    "large": {
+      "value": "1.375rem",
       "type": "paragraphSpacing"
     }
   },
-  "fontWeights": {
-    "headingRegular": {
-      "value": "Regular",
-      "type": "fontWeights"
-    },
-    "headingBold": {
+  "fontWeight": {
+    "bold": {
       "value": "Bold",
       "type": "fontWeights"
     },
-    "bodyRegular": {
-      "value": "Regular",
+    "semiBold": {
+      "value": "Semi Bold",
       "type": "fontWeights"
     },
-    "bodyBold": {
-      "value": "Bold",
+    "regular": {
+      "value": "Regular",
       "type": "fontWeights"
     }
   },
-  "fontSizes": {
-    "h1": {
-      "value": "{fontSizes.h2} * 1.25",
+  "fontSize": {
+    "headingXXXL": {
+      "value": "2.125rem",
       "type": "fontSizes"
     },
-    "h2": {
-      "value": "{fontSizes.h3} * 1.25",
+    "headingXXL": {
+      "value": "1.5rem",
       "type": "fontSizes"
     },
-    "h3": {
-      "value": "{fontSizes.h4} * 1.25",
+    "headingXL": {
+      "value": "1.25rem",
       "type": "fontSizes"
     },
-    "h4": {
-      "value": "{fontSizes.h5} * 1.25",
+    "headingL": {
+      "value": "1rem",
       "type": "fontSizes"
     },
-    "h5": {
-      "value": "{fontSizes.h6} * 1.25",
+    "headingM": {
+      "value": "0.85rem",
       "type": "fontSizes"
     },
-    "h6": {
-      "value": "{fontSizes.body} * 1",
+    "headingS": {
+      "value": "0.75rem",
       "type": "fontSizes"
     },
-    "body": {
-      "value": "16",
+    "bodyLarge": {
+      "value": "1rem",
       "type": "fontSizes"
     },
-    "sm": {
-      "value": "{fontSizes.body} * 0.85",
+    "bodyMedium": {
+      "value": "0.875rem",
       "type": "fontSizes"
     },
-    "xs": {
-      "value": "{fontSizes.body} * 0.65",
+    "bodySmall": {
+      "value": "0.75rem",
       "type": "fontSizes"
     }
+  },
+  "borderWidth": {
+    "default": {
+      "value": "1px",
+      "type": "borderWidth"
+    }
```

Comment:
In current MIUI we also have some bigger border width:
default: ['1px',
stroke: ['2px',
active: ['3px',
focus: ['4px',